deploy/openshift/template.yaml

```diff
@@ -1860,6 +1874,92 @@ objects:
                 httpGet:
                   path: /health/startup
                   port: 9010
+  - apiVersion: batch/v1
+    kind: CronJob
+    metadata:
+      name: v11y-walker
+      labels:
+        app.kubernetes.io/name: v11y-walker
+        app.kubernetes.io/component: walker
+        app.kubernetes.io/part-of: trustification
+    spec:
+      schedule: 0 1 * * *
+      concurrencyPolicy: Forbid
+      jobTemplate:
+        spec:
+          template:
+            metadata:
+              labels:
+                app.kubernetes.io/name: v11y-walker
+                app.kubernetes.io/component: walker
+                app.kubernetes.io/part-of: trustification
+            spec:
+              restartPolicy: Never
+              volumes:
+                - name: cvelist
+                  persistentVolumeClaim:
+                    claimName: v11y-cvelist
+              initContainers:
+                - name: git-fetch
+                  image: quay.io/trustification/git:2.40.1
+                  volumeMounts:
+                    - mountPath: /git
+                      name: cvelist
+                  command:
+                    - /bin/sh
+                  args:
+                    - -ec
+                    - "if test -d cvelistV5; then\n  cd cvelistV5\n  git pull\nelse\n\
```

Comment:
Any chance we can do the `|` thing and make the bash less \nescaped\n?

Reply:
This is auto-generated from here https://github.com/trustification/trustification/blob/main/deploy/k8s/chart/templates/v11y/walker/030-CronJob.yaml#L37-L43 - which is doing the | thing